rust/stackablectl/src/cli/mod.rs

```diff
@@ -185,7 +189,7 @@ pub enum OutputType {
 #[snafu(module)]
 pub enum CacheSettingsError {
     #[snafu(display("unable to resolve XDG directories"))]
-    Xdg { source: xdg::BaseDirectoriesError },
+    Xdg,
```

Comment:
Should we still call thisxdg, when xdg is only used in linux?

Reply:
Renamed in 1fe6c05